UI/multitrack-video-output.cpp

```diff
@@ -187,7 +187,9 @@ static void adjust_video_encoder_scaling(const obs_video_info &ovi, obs_encoder_
 
 	obs_encoder_set_scaled_size(video_encoder, requested_width, requested_height);
 	obs_encoder_set_gpu_scale_type(video_encoder, encoder_config.gpu_scale_type.value_or(OBS_SCALE_BICUBIC));
-	obs_encoder_set_preferred_video_format(video_encoder, VIDEO_FORMAT_NV12);
+	obs_encoder_set_preferred_video_format(video_encoder, encoder_config.video_format.value_or(VIDEO_FORMAT_NV12));
+	obs_encoder_set_preferred_color_space(video_encoder, encoder_config.video_colorspace.value_or(VIDEO_CS_709));
+	obs_encoder_set_preferred_range(video_encoder, encoder_config.video_range.value_or(VIDEO_RANGE_PARTIAL));
```

Comment:
Suggested by @palana to make this configurable as well in case somebody wants to do HDR on their platform.